our-initiatives/tutorials/intro-to-python.md

```diff
@@ -14,6 +14,13 @@ Here are the **exercises** you can go through: 📘 [**Tutorial 1 Exercises**](h
 
 You can access our **slides** here: 💻 [**Tutorial 1 Slides**](https://www.canva.com/design/DAFmvE-ptx0/lyY0SiOcjgSxrb201KcC8w/edit)
 
+The **recording** from this session is available here: 🎤 [**Tutorial 1 Recording**](https://www.youtube.com/watch?v=adnK-hc2DmY)
+
+Here are two videos covering our exercise **solutions**:
+
+- Exercises 1 to 4: [Solution pt1](https://www.youtube.com/watch?v=qmHgg0IR680&t=116s)
```

Comment:
I don't think this was intential (if it was, please ignore) but this link makes this video start 116 seconds after the beginning. Suggestion below!
```suggestion
- Exercises 1 to 4: [Solution pt1](https://www.youtube.com/watch?v=qmHgg0IR680)
```

Reply:
Oh yes my bad, i believe the correct link is: https://www.youtube.com/watch?v=qmHgg0IR680&t
(missed 2 char)